Dockerfile

```diff
@@ -1,27 +1,41 @@
-FROM node:10-buster-slim
-LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
+# Build stage 1.
+# Create base image with python (required for node-gyp)
 ARG BUILD_NUMBER
 ARG GIT_REF
 
+FROM node:10-buster-slim as base
```

Comment:
node 10?  Also don't think it is good to specify `buster` as that is the default anyway so we are pinning for no reason

Reply:
Upgrading from node 10 is on the backlog